### DISCUSSION TITLE
Failed to connect to server

### DISCUSSION THREAD

**Discussion by jbuczny** [asreview/asreview#1593] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
Dear All,

I cannot open any of my projects. I can view all the available projects, but when I attempt to any of them I get the prompt: "Failed to connect to server". I upgrade ASReview to the newest version – the behavior has not changed.

Any idea how to fix it?

Thanks,
Jacek

**Comment by J535D165** [comment]
What is the number of your port? Is it 5000? (localhost:5000)

Can you also show the output in the console?

**Comment by jbuczny** [comment]
Thank you for the swift reply.

Yes, my port is 5000.

Here is the console output:

---

Start browser at http://localhost:5000/

If your browser doesn't open. Please navigate to 'http://localhost:5000/'

127.0.0.1 - - [2023-11-22 15:49:03] "GET / HTTP/1.1" 200 713 0.016368
127.0.0.1 - - [2023-11-22 15:49:04] "GET /static/css/main.9e747be3.css HTTP/1.1" 304 192 0.012539
127.0.0.1 - - [2023-11-22 15:49:04] "GET /static/js/main.43c0da44.js HTTP/1.1" 200 1877819 0.278417
127.0.0.1 - - [2023-11-22 15:49:04] "GET /boot HTTP/1.1" 200 171 0.000387
127.0.0.1 - - [2023-11-22 15:49:04] "GET /boot HTTP/1.1" 200 171 0.000486
127.0.0.1 - - [2023-11-22 15:49:04] "GET /api/projects/stats HTTP/1.1" 200 164 0.024157
127.0.0.1 - - [2023-11-22 15:49:04] "GET /api/projects HTTP/1.1" 200 2025 0.001740
127.0.0.1 - - [2023-11-22 15:49:04] "GET /static/media/asreview_sub_logo_lab_black_transparent.c4e14af313ba97465fff113a6b2e4fd7.svg HTTP/1.1" 200 127815 0.003794
127.0.0.1 - - [2023-11-22 15:49:04] "GET /static/media/roboto-latin-400.176f8f5bd5f02b3abfcf.woff2 HTTP/1.1" 304 218 0.007340
127.0.0.1 - - [2023-11-22 15:49:04] "GET /static/media/roboto-latin-700.c18ee39fb002ad58b6dc.woff2 HTTP/1.1" 304 219 0.005985
127.0.0.1 - - [2023-11-22 15:49:04] "GET /static/media/roboto-latin-500.f5b74d7ffcdf85b9dd60.woff2 HTTP/1.1" 304 219 0.005838
127.0.0.1 - - [2023-11-22 15:49:04] "GET /favicon.ico HTTP/1.1" 304 185 0.001259
127.0.0.1 - - [2023-11-22 15:49:04] "GET /favicon.ico HTTP/1.1" 304 185 0.000987
zsh: segmentation fault  asreview lab

---

Shall I sent you anything else?

**Reply by tuotothesis** [reply]
Having the same problem! Did you figure out a solution in the end? Thank you.

**Reply by Rensvandeschoot** [reply]
It seems some of the dependencies for ASReview might require an update of your operating system. This worked for some of our users who experienced a similar issue. Can you let us know if it worked for you??

**Reply by itshudey** [reply]
Hi, I have the same problem when I want to create a project. The error message says, 'Failed to connect to server. Please restart the software.' What can I do to solve this problem?

**Reply by Rensvandeschoot** [reply]
@itshudey Can you copy-paste the information that is displayed in the command prompt?

**Reply by itshudey** [reply]
'Failed to connect to server. Please restart the software.'

**Reply by Rensvandeschoot** [reply]
Yes, this is the message you received in the webapp, but I am looking for the information in the command prompt (the screen in which you typed `asreview lab` to start the software).. thanks!

**Reply by itshudey** [reply]
(base) MacBook-Pro-van-Deniz:~ denizkarakus$ asreview lab
Start browser at http://localhost:5000/



If your browser doesn't open. Please navigate to 'http://localhost:5000/'




127.0.0.1 - - [2023-12-26 14:06:17] "GET / HTTP/1.1" 200 713 0.006806
127.0.0.1 - - [2023-12-26 14:06:17] "GET /static/css/main.9e747be3.css HTTP/1.1" 304 189 0.001039
127.0.0.1 - - [2023-12-26 14:06:17] "GET /static/js/main.43c0da44.js HTTP/1.1" 200 1877819 0.052588
127.0.0.1 - - [2023-12-26 14:06:17] "GET /boot HTTP/1.1" 200 171 0.000483
127.0.0.1 - - [2023-12-26 14:06:17] "GET /favicon.ico HTTP/1.1" 304 186 0.000634
127.0.0.1 - - [2023-12-26 14:06:17] "GET /boot HTTP/1.1" 200 171 0.000769
127.0.0.1 - - [2023-12-26 14:06:17] "GET /api/projects/stats HTTP/1.1" 200 164 0.001146
127.0.0.1 - - [2023-12-26 14:06:17] "GET /api/projects HTTP/1.1" 200 122 0.000599
127.0.0.1 - - [2023-12-26 14:06:18] "GET /static/media/asreview_sub_logo_lab_black_transparent.c4e14af313ba97465fff113a6b2e4fd7.svg HTTP/1.1" 304 252 0.000988
127.0.0.1 - - [2023-12-26 14:06:18] "GET /static/media/roboto-latin-400.176f8f5bd5f02b3abfcf.woff2 HTTP/1.1" 304 218 0.000827
127.0.0.1 - - [2023-12-26 14:06:18] "GET /static/media/roboto-latin-700.c18ee39fb002ad58b6dc.woff2 HTTP/1.1" 304 218 0.000714
127.0.0.1 - - [2023-12-26 14:06:18] "GET /static/media/roboto-latin-500.f5b74d7ffcdf85b9dd60.woff2 HTTP/1.1" 304 217 0.000936
127.0.0.1 - - [2023-12-26 14:06:18] "GET /static/media/ElasArrowRightAhead.e624bdd7c691c7c4f6c01604979c8f07.svg HTTP/1.1" 304 231 0.002072
Segmentation fault: 11
(base) MacBook-Pro-van-Deniz:~ denizkarakus$

**Reply by jbuczny** [reply, marked as answer]
Hi there!

I have just updated my mac from Big Sur to Sonoma. ASReview Lab works without any problems; I can access and resume my all my projects.

Best,
J
